Add flag that allows you to easily search all subdirectories (#210)
chef/license_scout: README.md

In addition to including any files Licensee identified as potential license files (but couldn't identify), License Scout will also include the Fallback License you specified in the Dependency Manifest.

### Searching Nested Subdirectories

> The default behavior of License Scout is to only look for dependency manager files in the root of the `directories` you configure. This is the default behavior to provide the maximum control over which dependencies you want to appear in your report. For example, your project may include an internal only tool for which you do not wish to enforce license acceptance.

Reviewer: ```suggestion
License Scout's default behavior is to only look for dependency manager files in the root of the `directories` that you configure. This default behavior provides greater control over the dependencies that you want to appear in your report. For example, you may not want to enforce license acceptance on an internal-only tool that is included in a project.
```

> However, License Scout also supports deep scanning all of the listed directories for all dependency manager files and generating a full report on all dependencies that the project uses. To do this, either specify the `--include-sub-directories` command line flag, or set `include_subdirectories` to true in your configuration file.

Reviewer: ```suggestion
License Scout will also scan subdirectories for all dependency manager files and generate a full report on all dependencies that the project uses. To do this, either specify the `--include-sub-directories` command line flag, or set `include_subdirectories` to true in your configuration file.
```
